docs/specs/oidc.md

```diff
@@ -300,24 +317,25 @@ To indicate the authenticator used in authentication, `amr` claim is used in OID
 - If password authenticator is used: `pwd` is included.
 - If any OTP (TOTP/OOB-OTP) is used: `otp` is included.
 - If WebAuthn is used: `hwk` is included.
+- If Biometric is used: `x_biometric` is included.
 
-If no authentication method is to be included in `amr` claim, `amr` claim would be omitted from the ID token.
+If no authentication method is included in `amr` claim, `amr` claim would be omitted from the ID token.
 
-### `acr`
+### `auth_time`
 
-If any secondary authenticator is performed, `acr` claim would be included in ID token with value `http://schemas.openid.net/pape/policies/2007/06/multi-factor`.
+The time when the authentication was performed.
+This claim is only present if the ID token is obtained via the authorization code flow or the implicit flow.
 
-To perform step-up authentication, developer can pass a `acr_values` of  `http://schemas.openid.net/pape/policies/2007/06/multi-factor` to the authorize endpoint.
+This is the authenticated at of the IdP session.
```

Comment:
Question: do our current design allow step-up authentication in parallel with re-auth?

Reply:
We now have `auth_time` and `amr` in the ID token. From these two claims, we can know **when** was the last authentication done (from `auth_time`) and **how** was it done (from `amr`). In the future, we can add our own parameters to the authorization endpoint to specify what kind of authentication the developer wants. When we add such option, the default is re-auth with any authenticator (i.e. the current behavior)

Reply:
Got it